cardigan/fractal-app.js

```diff
@@ -3,20 +3,21 @@ import Fractal from '@frctl/fractal';
 import Nunjucks from '@frctl/nunjucks';
 import mandelbrot from '@frctl/mandelbrot';
 import {Map} from 'immutable';
-import filters from './filters';
-import extensions from './extensions';
-import {getEnvWithGlobalsExtensionsAndFilters} from './view/env-utils';
-import {createPageConfig} from './model/page-config';
+import filters from '../server/filters';
```

Comment:
We could look at potentially `npm link`ing these, but felt the less I move, the better for the first PR.